pages/en/tokenomics.mdx

```diff
@@ -0,0 +1,102 @@
+
+As of December 2022, subgraph developers are encouraged to curate their subgraph with at least 10,000 GRT. However, this number may be impacted by network activity and community participation.
+
+Curators pay a 1% curation tax when they curate a new subgraph. This curation tax is burned, decreasing the supply of GRT.
+
+# Developers
+
+Developers build and query subgraphs to retrieve blockchain data. Since subgraphs are open-source, developers can query existing subgraphs to load blockchain data into their dapps. Developers pay for queries they make in GRT, that is distributed to network participants.
+
+## Creating a subgraph
+
+Developers can [create a subgraph](https://thegraph.com/docs/en/developing/creating-a-subgraph/) to index data on the blockchain. Subgraphs are instructions for Indexers about which data should be served to consumers.
+
+Once developers have built and tested their subgraph, they can [publish their subgraph](https://thegraph.com/docs/en/publishing/publishing-a-subgraph/) on The Graph's decentralized network.
+
+## Querying an existing subgraph
+
+Once a subgraph is [published](https://thegraph.com/docs/en/publishing/publishing-a-subgraph/) to The Graph's decentralized network, anyone can create an API key, fill it with GRT, and query it.
+
+Subgraphs are [queried using GraphQL](https://thegraph.com/docs/en/querying/querying-the-graph/), and the query fees are paid for with GRT in [Subgraph Studio](https://thegraph.com/studio/). Query fees are distributed to network participants based on their contributions to the protocol.
+
+1% of the query fees paid to the network are burned.
+
+# Indexers (Earn GRT)
+
+Indexers are the backbone of The Graph. They operate independent hardware and software to power the network. Indexers serve data to consumers based on instructions from subgraphs.
+
+Indexers can earn GRT rewards in two ways:
+
+1.  Query fees: GRT paid by developers or users for subgraph data queries. Query fees are deposited into a rebate pool and distributed to Indexers.
+
+2.  Indexing rewards: the 3% annual issuance is distributed to Indexers based on the number of subgraphs they are indexing. Indexing rewards incentivize Indexers to index subgraphs, sometimes before the query fees begin to accrue, and to submit Proofs of Indexing (POIs), which verify that they have indexed data accurately.
+
+Each subgraph is allotted a portion of the total network token issuance, based on the amount of curation signal that subgraph has. That amount is then divided between all the Indexers staked on that subgraph based on their allocated stake.
```

Comment:
"divided between all the Indexers staked" reads a bit funny to me.

Reply:
Reworded it! Good call!